.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: perl
+perl:
+  - "5.30"
+  - "5.28"
+  - "5.26"
+  - "5.24"
+  - "5.22"
+  - "5.20"
+  - "5.18"
+  - "5.16"
+  - "5.30-shrplib"
+  - "5.28-shrplib"
+  - "5.26-shrplib"
+  - "5.24-shrplib"
+  - "5.22-shrplib"
+  - "5.20-shrplib"
+  - "5.18-shrplib"
+notifications:
+  email:
+    recipients:
+      - jkeenan@cpan.org
```

Comment:
I don't think this should remain in the master version.  If the notification section is left out, by default it is always the commit owner who is notified, which should be sufficient in most cases.

Reply:
At this point, feel free to make any appropriate revisions in either `.travis.yml` or `.appveyor.yml`. As long as there are such files (which I provided in response to mohawk's suggestion), I'm happy.

Reply:
Well if you're happy getting notified for every travis failure going forward, have fun I guess :)